packages/playground/src/components/dapp-container/dapp-container.tsx

```diff
@@ -106,7 +108,7 @@ export class DappContainer {
   }
 
   private async handlePlaygroundMessage(event: MessageEvent): Promise<void> {
-    if (!this.frameWindow) {
+    if (!this.frameWindow || typeof event.data !== "string") {
```

Comment:
What does `typeof event.data !== "string"` represent? Maybe assign it to a variable with a descriptive name

Reply:
event.data is the postMessage() payload sent from the dApp to the playground. Since the `message` event is also shared with browser extensions and all sort of things that *could* be transferring objects, I need to type check it.